x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.tsx

```diff
@@ -135,13 +135,15 @@ export const InferencePipelineCard: React.FC<InferencePipeline> = ({
                     <EuiHealth color="success">{deployedText}</EuiHealth>
                   </EuiFlexItem>
                 )}
-                <EuiFlexItem grow={false}>
-                  <EuiFlexGroup gutterSize="xs">
-                    <EuiFlexItem>
-                      <EuiBadge color="hollow">{modelType}</EuiBadge>
-                    </EuiFlexItem>
-                  </EuiFlexGroup>
-                </EuiFlexItem>
+                {types.map((type) => (
+                  <EuiFlexItem grow={false}>
```

Comment:
This would need a unique `key` for each or react will give a warning.

I _think_ there is a component we can use that will auto truncate when there are too many badges? I'll see if I can find an example

Reply:
[EuiBadgeGroup](https://elastic.github.io/eui/#/display/badge#badge-groups-and-truncation) ?

Reply:
> This would need a unique `key` for each or react will give a warning.

Great call! I forgot to run the test file for this file locally. It indeed failed 👍 

> I _think_ there is a component we can use that will auto truncate when there are too many badges? I'll see if I can find an example

Would be very helpful 🙌